src/Arbus.Network/HttpContentType.cs

```diff
@@ -17,4 +16,6 @@ public static class Text
     }
 
     public const string ImageSvgXmlContentType = "image/svg+xml";
+    
+    public const string MultipartFormData = "multipart/form-data";
```

Comment:
I believe we better separate class for multipart content-type as we did with Application.cs above.
Multipart content-type has four different values: [All possible values of HTTP Content-type header](https://www.geeksforgeeks.org/http-headers-content-type/)

Reply:
Should I add all 4 multipart content types or we will be sufficient by this one?

Reply:
That's a great idea! As Arbus.Network is a public package and anyone might use it for any purpose